pallets/gear/src/lib.rs

```diff
@@ -198,14 +198,22 @@ pub mod pallet {
         >;
 
         /// Implementation of a ledger to account for gas creation and consumption
-        type GasProvider: common::GasProvider<
+        type GasProvider: GasProvider<
             ExternalOrigin = Self::AccountId,
             Key = MessageId,
             Balance = u64,
             Error = DispatchError,
         >;
 
-        type BlockLimiter: common::BlockLimiter<Balance = u64>;
+        /// Block limits.
+        type BlockLimiter: BlockLimiter<Balance = u64>;
+
+        /// Scheduler.
```

Comment:
Lets leave them as it is, because more informative docs are in common crate, but here in-place its like placeholder. Anyway before publish on crates io we will walk through our docs with proofread